sale_order_line_position/models/ir_actions_report.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2021 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+from odoo import models
+
+
+class IrActionsReport(models.Model):
+    _inherit = "ir.actions.report"
+
+    def render_qweb_pdf(self, res_ids=None, data=None):
```

Comment:
it should be `def _render_qweb_pdf()`. btw, you must make sure the positions are recomputed when printing the Quotation / Order report.

Reply:
it's better if you create a unit test for this. so, we don't miss this point next time of the migration.